prov/efa/src/efa_base_ep.c

```diff
@@ -79,6 +70,20 @@ int efa_base_ep_destruct(struct efa_base_ep *base_ep)
 	return err;
 }
 
+int efa_base_ep_close(struct efa_base_ep *base_ep)
```

Comment:
This basically just close util ep, I would just name it `efa_base_ep_close_util_ep`

Reply:
done